apps/portal/src/content/docs/components/avatar.mdx

```diff
@@ -4,150 +4,171 @@ description: Avatar component
-  code={`<div className="flex flex-col gap-8">
-    <div className="flex flex-col items-center gap-4">
-        <h5>With image</h5>
-        <Avatar.Root size={8}>
-            <Avatar.Image src="https://github.com/shadcn.png" alt="Avatar image" />
-            <Avatar.Fallback>JD</Avatar.Fallback>
-        </Avatar.Root>
-    </div>
-    <hr />
-    <div className="flex flex-col items-center gap-4">
-        <h5>With fallback</h5>
-        <Avatar.Root size={8}>
-            <Avatar.Fallback>JD</Avatar.Fallback>
-        </Avatar.Root>
-    </div>
+  code={`<div className="flex gap-12">
+  <div className="flex flex-col gap-3 items-center">
+    <h6 className="text-sm">Default Size</h6>
+    <Avatar name="John Doe" src="https://github.com/shadcn.png" />
+  </div>
+
+<div className="flex flex-col gap-3 items-center">
+  <h6 className="text-sm">Large Size</h6>
+  <Avatar name="John Doe" src="https://github.com/shadcn.png" size="lg" />
+</div>
+
+  <div className="flex flex-col gap-3 items-center">
+    <h6 className="text-sm">Small Size</h6>
+    <Avatar name="John Doe" src="https://github.com/shadcn.png" size="sm" />
+  </div>
 </div>`}
 />
 
-## Usage
+### Image (Rounded)
 
-```typescript jsx
-import { Avatar } from '@harnessio/ui/components'
+<DocsPage.ComponentExample
+  client:only
+  code={`<div className="flex gap-12">
+  <div className="flex flex-col gap-3 items-center">
+    <h6 className="text-sm">Default Size</h6>
+    <Avatar name="John Doe" src="https://github.com/shadcn.png" rounded />
+  </div>
+
+<div className="flex flex-col gap-3 items-center">
+  <h6 className="text-sm">Large Size</h6>
+  <Avatar
+    name="John Doe"
+    src="https://github.com/shadcn.png"
+    size="lg"
+    rounded
+  />
+</div>
+
+  <div className="flex flex-col gap-3 items-center">
+    <h6 className="text-sm">Small Size</h6>
+    <Avatar name="John Doe" src="https://github.com/shadcn.png" size="sm" rounded />
+  </div>
+</div>`}
+/>
 
-//...
+### Initials (Default)
 
-return (
-  <Avatar.Root>
-    <Avatar.Image src="avatarurl.jpg" alt="Avatar image" />
-    <Avatar.Fallback>JD</Avatar.Fallback>
-  </Avatar.Root>
-)
-```
+<DocsPage.ComponentExample
+  client:only
+  code={`<div className="flex gap-12">
+  <div className="flex flex-col gap-3 items-center">
+    <h6 className="text-sm">Default Size</h6>
+    <Avatar name="John Doe" />
+  </div>
+
+<div className="flex flex-col gap-3 items-center">
+  <h6 className="text-sm">Large Size</h6>
+  <Avatar name="John Doe" size="lg" />
+</div>
+</div>`}
+/>
+
+### Initials (Rounded)
+
+<DocsPage.ComponentExample
+  client:only
+  code={`<div className="flex gap-12">
+  <div className="flex flex-col gap-3 items-center">
+    <h6 className="text-sm">Default Size</h6>
+    <Avatar name="John Doe" rounded />
+  </div>
+
+<div className="flex flex-col gap-3 items-center">
+  <h6 className="text-sm">Large Size</h6>
+  <Avatar name="John Doe" size="lg" rounded />
+</div>
+</div>`}
+/>
 
-## Anatomy
+## Usage
```

Comment:
Can we revise the usage section to bring it in line with the other component docs?